packages/riipen-ui/src/components/GridItem.jsx

```diff
@@ -43,18 +48,21 @@ class GridItem extends React.Component {
   static defaultProps = {
     classes: [],
     lg: 12,
-    spacing: 3
+    spacing: 3,
+    flexGrow: false
```

Comment:
Why not have a default of 0 here. That way it's more inline with the actual allowed values of `flex-grow`.